### PR TITLE
added colors in taiwind config and made changes required

### PR DESCRIPTION
**Pull request by chefinazia**
image

New looks like this...

image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -3,11 +3,11 @@ import LoginButton from "./components/LoginButton";
...
 const App = () => {
   return (
-    <div className="grid grid-rows-3 grid-flow-col gap-4">
-      <div className="row-span-3">
+    <div className="divide-y-4">
+      <div className="">
         <LoginButton />
       </div>
-      <div className="col-span-2">
+      <div className="">
```
Comment (Deepak-Bulani) [inline src/App.jsx:10]: if class doesn't exist, remove className

**Review by Deepak-Bulani — CHANGES_REQUESTED** [review]

File: src/components/CalendarPatientsList.jsx [hunk 2]
```diff
@@ -13,44 +13,41 @@ const CalendarPatientsList = () => {
...
     <div className="grid">
-      <div className="col-span-2 flex">
-        <div
-          className="flex flex-initial align-middle justify-center none border w-auto border-zinc-800 center rounded-lg font-serif text-md text-center uppercase text-zinc-700 py-3"
-          data-ripple-dark="true"
+        <button
+          className="flex divide-x-3 m-auto max-w-fit px-4 justify-center align-middle bg-primary-secondaryTabColor none border border-primary-secondaryTabColorBorderColor center rounded-md font-serif text-md text-center uppercase text-primary-textColor py-3"
         >
-          <div className="flex-initial">
+          <div className="grid-rows-2 gap-4">
```
[the remark this answers is not in the document]
Comment (chefinazia) [reply src/components/CalendarPatientsList.jsx:19]: replaced with div

File: src/App.jsx (hunk 1 above)
Comment (chefinazia) [reply src/App.jsx:10]: removed

File: src/components/CalendarPatientsList.jsx [hunk 3]
```diff
@@ -13,44 +13,41 @@ const CalendarPatientsList = () => {
 
   return (
     <div className="grid">
-      <div className="col-span-2 flex">
-        <div
-          className="flex flex-initial align-middle justify-center none border w-auto border-zinc-800 center rounded-lg font-serif text-md text-center uppercase text-zinc-700 py-3"
-          data-ripple-dark="true"
+        <button
+          className="flex divide-x-3 m-auto max-w-fit px-4 justify-center align-middle bg-primary-secondaryTabColor none border border-primary-secondaryTabColorBorderColor center rounded-md font-serif text-md text-center uppercase text-primary-textColor py-3"
```
[the remark this answers is not in the document]
Comment (chefinazia) [reply src/components/CalendarPatientsList.jsx:17]: okay removed

File: src/components/CalendarPatientsList.jsx [hunk 4]
```diff
@@ -12,45 +14,28 @@ const CalendarPatientsList = () => {
...
-              <svg
-                className="w-6 h-6 ml-2"
-                viewBox="0 0 24 24"
-                fill="none"
-                xmlns="http://www.w3.org/2000/svg"
-              >
-                <path d="M10 7l6 5-6 5V7z" fill="#0D0D0D" />
-              </svg>
+              <CaretRight/>
             </a>
```
Comment (kanayan007) [inline src/components/CalendarPatientsList.jsx:35]: Avoid using Anchor tag for achieving some functionality based on onClick. Anchor tag will be used for navigation and does a full page reload unless you use preventDefault(). So, for navigation also we should be using Link tag from react-router-dom

**Review by kanayan007 — CHANGES_REQUESTED** [review]

**Pull request merged**